CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## UNRELEASED
+
+- Fixed return type of `generate` so that it is consistent with the possibility of returning a `string` rather than a `string[]`
```

Comment:
Small, nit-picky change. There is a possibility of returning a `string` or a `string[]`.

Reply:
Yep, could've used better wording on my part. Updated to read:
```markdown
- Fixed return type of `generate` so that it is consistent with the possibility of returning either `string` or `string[]`
```

Reply:
Thanks for this change!